recipes/lixtools/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lixtools" %}
+{% set version = "2021.6.9.0" %}
+{% set sha256 = "d12e77bfc770ddb9137345c054a6c02afba29bac72768212615126b94b062c4b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
+    - numpy
+    - openpyxl >=3
+    - pandas
+    - pillow
+    - py4xs
+    - python-barcode
+    - qrcode
+
+test:
+  imports:
+    - lixtools
```

Comment:
Can you add some better import tests (`__init__` only sets the version)? I see a whole bunch of other imports in other modules that aren't listed above, like dask and scipy.